rs/registry/canister/src/registry.rs

```diff
@@ -488,12 +488,17 @@ impl Registry {
                     }
                     // End code to fix ICSUP-2589
 
-                    let mutation = HighCapacityRegistryAtomicMutateRequest::decode(
+                    let mut mutation = HighCapacityRegistryAtomicMutateRequest::decode(
                         &entry.encoded_mutation[..],
                     )
                     .unwrap_or_else(|err| {
                         panic!("Failed to decode mutation@{}: {}", entry.version, err)
                     });
+                    // TODO: Remove this once the timestamps have been converted to seconds.
```

Comment:
Can you create a Jira ticket, and reference it like so:

```
TODO(${JIRA_TICKET_ID})
```

I thought this was a DFINITY convention, but I maybe am just imagining that. Either way, it's a good thing to do, because it (somewhat?) helps ensure that we eventually do it.

Reply:
Absolutely! I've seen your team maintain that practice but I didn't really look into other bits of code.